resources/views/checkout/partials/summary.blade.php

```diff
@@ -0,0 +1,116 @@
+    </x-rapidez::summary.row>
+
+    <x-rapidez::summary.divider />
+
+    <x-rapidez::summary.row>
+        <template v-if="cart.shipping_addresses[0]">
+            <x-rapidez::summary.title>
+                <template v-if="cart.billing_address?.same_as_shipping">@lang('Shipping & billing address')</template>
+                <template v-else>@lang('Shipping address')</template>
+            </x-rapidez::summary.title>
+            <x-rapidez::summary.description>
+                <ul>
+                    <li v-if="cart.shipping_addresses[0]?.company">@{{ cart.shipping_addresses[0]?.company }}</li>
+                    <li v-if="cart.shipping_addresses[0]?.prefix || cart.shipping_addresses[0]?.firstname || cart.shipping_addresses[0]?.middlename || cart.shipping_addresses[0]?.lastname || cart.shipping_addresses[0]?.suffix">@{{ cart.shipping_addresses[0]?.prefix }} @{{ cart.shipping_addresses[0]?.firstname }} @{{ cart.shipping_addresses[0]?.middlename }} @{{ cart.shipping_addresses[0]?.lastname }} @{{ cart.shipping_addresses[0]?.suffix }}</li>
+                    <li v-if="cart.shipping_addresses[0]?.street[0] || cart.shipping_addresses[0]?.street[1] || cart.shipping_addresses[0]?.street[2]">@{{ cart.shipping_addresses[0]?.street[0] }} @{{ cart.shipping_addresses[0]?.street[1] }} @{{ cart.shipping_addresses[0]?.street[2] }}</li>
+                    <li v-if="cart.shipping_addresses[0]?.postcode || cart.shipping_addresses[0]?.city || cart.shipping_addresses[0]?.country.label">@{{ cart.shipping_addresses[0]?.postcode }} - @{{ cart.shipping_addresses[0]?.city }} - @{{ cart.shipping_addresses[0]?.country.label }}</li>
+                    <li v-if="cart.shipping_addresses[0]?.telephone">@{{ cart.shipping_addresses[0]?.telephone }}</li>
+                </ul>
+            </x-rapidez::summary.description>
+        </template>
+        <template v-if="cart.billing_address && !cart.billing_address?.same_as_shipping">
+            <x-rapidez::summary.divider />
+            <x-rapidez::summary.title>
+                @lang('Billing address')
+            </x-rapidez::summary.title>
+            <x-rapidez::summary.description>
+                <ul>
+                    <li v-if="cart.billing_address.company">@{{ cart.billing_address.company }}</li>
+                    <li>@{{ cart.billing_address.prefix }} @{{ cart.billing_address.firstname }} @{{ cart.billing_address.middlename }} @{{ cart.billing_address.lastname }} @{{ cart.billing_address.suffix }}</li>
+                    <li>@{{ cart.billing_address.street[0] }} @{{ cart.billing_address.street[1] }} @{{ cart.billing_address.street[2] }}</li>
+                    <li>@{{ cart.billing_address.postcode }} - @{{ cart.billing_address.city }} - @{{ cart.billing_address.country.label }}</li>
+                    <li v-if="cart.billing_address.telephone">@{{ cart.billing_address.telephone }}</li>
+                </ul>
+            </x-rapidez::summary.description>
```

Comment:
Lots of duplication here for 2 addresses, can we fix that? Also missing some v-ifs on the billing-address side currently.